src/Flex.php

```diff
@@ -231,6 +232,15 @@ function ($value) {
             }
         }
 
+        if (null !== $manifest) {
+            array_unshift(
+                $this->postInstallOutput,
+                '',
+                '<info>A few files may have been created to configure newly installed packages.</>',
```

Comment:
Some files may have been created or updated to configure your newly installed packages.